noise_delta_t: 1.0
map_frame_id: 'map'
odom_frame_id: 'zed_objdet_odom'
field_dims:
  x_min: 0.0
  x_max: 6.0
  y_min: 0.0
  y_max: 6.0

init_dims:
  x_min: 0.0
  x_max: 6.0
  y_min: 0.0
  y_max: 6.0

num_particles: 100

noise_stdev:
  position: 0.01
  rotation: 0.01

beacons:
  - [0.24, 0.0, "2"]
  - [0.95, 0.0, "0"]
  - [1.57, 0.68, "1"]
